WalletWasabi/Rpc/WasabiJsonRpcService.cs

```diff
@@ -5,25 +5,37 @@
 using WalletWasabi.BitcoinP2p;
 using WalletWasabi.Blockchain.Analysis.Clustering;
 using WalletWasabi.Blockchain.Keys;
+using WalletWasabi.Blockchain.TransactionBroadcasting;
 using WalletWasabi.Blockchain.TransactionBuilding;
 using WalletWasabi.Blockchain.Transactions;
 using WalletWasabi.CoinJoin.Client.Clients.Queuing;
 using WalletWasabi.Helpers;
 using WalletWasabi.Models;
+using WalletWasabi.Services;
 using WalletWasabi.Services.Terminate;
+using WalletWasabi.Stores;
 using WalletWasabi.Wallets;
 
-namespace WalletWasabi.Fluent.Rpc
+namespace WalletWasabi.Rpc
 {
 	public partial class WasabiJsonRpcService
 	{
-		public WasabiJsonRpcService(Global global, TerminateService terminateService)
+		#region Dependencies
+
+		public WalletManager WalletManager { get; set; }
```

Comment:
Can you add compulsory dependencies to the ctor?

Reply:
no because it goes against unit testing. You want to be able to omit anything but the dependencies that are necessary for given test. Not all methods require WalletManager

Reply:
The dependencies can be injected in the ctor or one by one at the methods. It has to be clearly defined what is required for the service in advance. Tests are secondary. If absolutely required tests can be mocked. Also, we are fully engaged with Nullable considerations as well.